labs/environment_spatial2022.yml

```diff
@@ -48,17 +33,13 @@ dependencies:
   - squidpy>=1.2.2
   - morphops
   - napari
+  - napari-tissuumaps
```

Comment:
napari-tissuumaps is only a plugin for napari but does not contain the TissUUmaps software. Unfortunately TissUUmaps doesn't have a conda version yet.

Reply:
Ahh, I did not pay enough attention. Here’s a PR for a TissUUmaps recipe: https://github.com/bioconda/bioconda-recipes/pull/43282 It does not contain the PyQt6 GUI as there’s no pyqt6 available for Conda. Would that be good enough for the course?

Reply:
I’ve removed this particular change.